lib/action_controller/serialization.rb

```diff
@@ -13,6 +13,11 @@ def serialization_scope(scope)
       end
     end
 
+    class << self
+      attr_accessor :enabled
+    end
+    self.enabled = true
```

Comment:
(pardon the awful pseudo code)

I was actually just thinking about having 
- a configurable default 'ams_ify_controller = true` 
- and then in the controller, by default we only mix in a 'shim' module that
  - has config methods, e.g. ams_ify_controller
  - so that by default ActionController::Serialization would be included in action controller
  - but if the default were set to false, then individual controllers would need to opt in by setting `ams_ify_controller true` .    
  - the method would just extend the controller  with ActionController::Serialization when true